backend/controller/ingress.go

```diff
@@ -0,0 +1,67 @@
+package controller
+
+import (
+	"context"
+	"math/rand"
+	"strings"
+
+	"github.com/alecthomas/errors"
+
+	"github.com/TBD54566975/ftl/backend/common/slices"
+	"github.com/TBD54566975/ftl/backend/controller/dal"
+)
+
+func (s *Service) getIngressRoute(ctx context.Context, method string, path string) (*dal.IngressRoute, error) {
+	pathStart := strings.Split(path, "/")[1]
```

Comment:
Won't this be invalid if the user has something like `/{foo}`, or does `path` include the `/ingress` prefix?